buildLauncher.ps1

```
@@ -1,273 +1,223 @@
+        [Parameter(Mandatory = $true)]
+        [string]$DestinationZip
+    )
+
+    # 确保目标ZIP文件的目录存在
+    # $destinationDir = [System.IO.Path]::GetDirectoryName($DestinationZip)
+    # CheckDir -Dir $DestinationZip;
+
+    # 打包文件和文件夹
+    try {
+        Compress-Archive -Path $SourceItems -DestinationPath $DestinationZip -Force
+        Write-Host "打包成功！输出文件：${DestinationZip}" -ForegroundColor Green
```

PSScriptAnalyzer (1.25.0)

[warning] File 'buildLauncher.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)